### PR TITLE
Userstory3

### PR DESCRIPTION
**Pull request by falkkreimeier**

### CODE REVIEW

File: src/OrderingForm.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+    <Wrapper>
+      <Headline>Rette mich, wer kann!</Headline>
+      <Form>
+        <InputContainer>
+          <Label>
+            <HiddenLabelText>Vorname:</HiddenLabelText>
+            <Vorname
+              type="text"
+              placeholder="Vorname"
+              name="nachname"
```
Comment (lukemt) [inline src/OrderingForm.js:14]: Vorname != nachname 😃 
@falkkreimeier

**Review by lukemt — COMMENTED** [review]
i found something

File: src/GlobalStyles.js [hunk 2]
```diff
@@ -20,7 +25,7 @@ body {
 }
 
 h2 {
-    font-family: Wisdom;
+    font-family: Ventana;
```
Comment (macfrei) [inline src/GlobalStyles.js:28]: Same here.

File: src/OrderingForm.js [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+    'straße'
+    'postleitzahl'
+    'stadt'
+    'email'
+    'menge'
+    'message'
+    'button';
+`
+
+const Label = styled.label``
```
Comment (macfrei) [inline src/OrderingForm.js:127]: You are not adding any styles to this `Label` component, so you can just use the normal HTML `label` element.

File: src/OrderingForm.js [hunk 4]
```diff
@@ -0,0 +1,240 @@
+import styled, { css } from 'styled-components'
+
+function OrderingForm() {
+  return (
+    <Wrapper>
+      <Headline>Rette mich, wer kann!</Headline>
+      <Form>
```
Comment (macfrei) [inline src/OrderingForm.js:7]: Why didn't you add a `submit` event? With not adding it, this component is not really functional as yet.

File: src/ProductCard.js [hunk 5]
```diff
@@ -139,28 +146,29 @@ const ButtonBack = styled.button`
...
-const Details = styled.ul`
+const Details = styled.ol`
+  margin: -10px 15px;
   grid-area: details;
 `
 
 const List = styled.li`
   width: 260px;
-  list-style: none;
+  /* list-style: none; */
```
Comment (macfrei) [inline src/ProductCard.js:171]: I you don't need it, please delete it!

File: src/ProductCard.js [hunk 6]
```diff
@@ -139,28 +146,29 @@ const ButtonBack = styled.button`
...
+  margin: -40px -10px; ;
 `
 
-const Details = styled.ul`
+const Details = styled.ol`
+  margin: -10px 15px;
   grid-area: details;
 `
 
 const List = styled.li`
```
Comment (macfrei) [inline src/ProductCard.js:169]: This should not be called `List` as it is not a list but a list item. So please call it `ListItem` or something like that.

**Pull request merged**